calculator.js

```diff
@@ -0,0 +1,30 @@
+const calculator = (() => ({
+  add: (...args) => args.reduce((cr, acc) => {
```

Comment:
Just the names, the accumulator is the first argument passed by `reduce`, the current value the second.

Reply:
Sorry, can you ellaborate on that.